src/main/java/com/github/tomakehurst/wiremock/standalone/JsonFileMappingsSource.java

```diff
@@ -87,6 +88,14 @@ public void loadMappingsInto(StubMappings stubMappings) {
 	}
 
 	private String getFileName(TextFile mappingFile) {
-		return mappingFile.getUri().toString().replaceAll("^.*/", "");
+		URI mappingFileUri = mappingFile.getUri();
```

Comment:
I considered moving this logic to a method in `FileSource` (called something like `relativePath()`), but `FileSource` is a public interface so I figured we shouldn't break BC for this.

Reply:
Adding new methods is fine. It's just renaming, removing or altering the signature of existing ones that causes BC problems. If you think this would be a preferable way to do it, I'd say go ahead.